src/pages/whats-happening/releases/index.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Releases
+description:
+  Read about the latest updates and learn more about what the team has planned
+  for Carbon’s future.
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListRow,
+  StructuredListCell,
+  StructuredListBody,
+  Link,
+  UnorderedList,
+} from 'carbon-components-react';
+
+<PageDescription>
+
+Read about the latest updates and learn more about what the team has planned for
+Carbon’s future.
+
+</PageDescription>
+
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Changelog"
+    href="https://github.com/carbon-design-system/carbon/releases">
+    <MdxIcon name="github" />
+  </ResourceCard>
+</Column>
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Roadmap"
+    href="https://app.zenhub.com/workspaces/system-squad-593830641344b813db10934d/roadmap">
+
+![ZenHub icon](images/z-icon.svg)
+
+  </ResourceCard>
+</Column>
+</Row>
+
+## Releases
+
+Each Carbon Design System release includes package releases that are
+independently versioned following the industry-standard
+[semantic versioning](https://semver.org) specification.
+
+The Carbon team also ships package minor releases every other week and when
+necessary, patch releases between minor versions to address bugs. Read more in
+Carbon's
+[release radar](https://github.com/carbon-design-system/carbon/wiki/Release-radar).
+
+### Carbon TBD
+
+Carbon TBD uses newer browser capabilities and includes quality of life updates
+for improved designer and developer experiences. As v2 of the IBM Design
+Language is still current, Carbon TBD will not require brand-driven product
+redesigns. Read more about the upcoming
+[Carbon TBD release](https://medium.com/carbondesign/whats-coming-to-carbon-in-2021-39a4c7d1762a).
+
+### Carbon v10
+
+Carbon v10 is the digital expression of the IBM Design Language, v2. The updated
```

Comment:
From @mjabbink:

something along these lines

(from)

Carbon v10 is the digital expression of the IBM Design Language, v2. The updated components required some product redesigns, however they had a clearly identifiable IBMness and created a connective tissue between all product experiences.

(to)

Carbon v10 was the release that implemented our newly updated IBM brand expression (Design Language v2) into our components. The updated components required product teams to redesign product UIs and begin a plan to migrate and/or adopt the new components to copy to IBM brand experience standards.

Reply:
@mattrosno Suggested tweak to this: 

Carbon v10 implemented our newly updated IBM brand expression (Design Language v2) into our components. The updated components required product teams to redesign product UIs and begin a plan to migrate and/or adopt the new components to comply with IBM brand experience standards.

Reply:
Done!